tools/Redirector/config.txt

```diff
@@ -1,12 +0,0 @@
-[SERVERS]
-## Simply enter a list of servers to poll. Be sure to specify a server name in parentheses.
-
-(Sibyl #1) byond://game.nanotrasen.com:1337
-
-(Sibyl #2) byond://game.nanotrasen.com:2337
```

Comment:
what if i told you a new contributor really tried running the server from redirector and it sent them to one of these

Reply:
unironically this, i had them check their hosts file and everything because I was so confused on how they were getting sent to the old SoS server before remembering the redirector existed

Reply:
lmao based